core/templates/cards.html

```diff
@@ -234,7 +234,7 @@ <h4 class="modal-title">Face Off</h4>
                 card.attr('onclick', 'flipNoScore(' + index + ')');
             }
         }
-        if (gameMode != easyGame) {
+        if (gameMode != easyGame && round != 4) {
             $("#countdown").hide();
```

Comment:
just out of curiosity what was the case this was failing? so we can make sure we add it as a testcase